<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trygonometryczne</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ikona.png</normaloff>ikona.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj ilość stopni początkowego kąta:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textKatA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textKatB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>360</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj ilość stopni końcowego kąta:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textPodzialy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj ilość podziałów pomiędzy kątami:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushOblicz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz i generuj wykres</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>701</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyniki:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wykres z wynikami:</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>1011</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sinus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cosinus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tanges</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cotanges</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj wykres dla wartości:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrygonometryczne">
    <property name="title">
     <string>Men&amp;u</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionEksportuj_wyniki"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuTrygonometryczne_2">
    <property name="title">
     <string>Pomo&amp;c</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionO_Qt"/>
   </widget>
   <addaction name="menuTrygonometryczne"/>
   <addaction name="menuTrygonometryczne_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionO_programie">
   <property name="text">
    <string>&amp;O programie</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>&amp;Zamknij</string>
   </property>
  </action>
  <action name="actionO_Qt">
   <property name="text">
    <string>O &amp;Qt</string>
   </property>
  </action>
  <action name="actionEksportuj_wyniki">
   <property name="text">
    <string>Eksportuj wyniki</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
